clients/admin-ui/src/features/privacy-requests/DownloadAccessResults.tsx

```diff
@@ -0,0 +1,52 @@
+import { AntButton as Button, Flex, Tooltip } from "fidesui";
+
+import { getErrorMessage } from "~/features/common/helpers";
+import { DownloadLightIcon } from "~/features/common/Icon";
+import Restrict from "~/features/common/Restrict";
+import { useGetPrivacyRequestAccessResultsQuery } from "~/features/privacy-requests/privacy-requests.slice";
+import { ScopeRegistryEnum } from "~/types/api";
+
+const DownloadAccessResults = ({ requestId }: { requestId: string }) => {
+  const { data, isLoading, isError, error } =
+    useGetPrivacyRequestAccessResultsQuery({
+      privacy_request_id: requestId,
+    });
+
+  const downloadButtonTooltip = isError
+    ? getErrorMessage(error, "Unable to resolve download URL")
+    : "Requests stored locally cannot be downloaded";
```

Comment:
Requests shouldn't ever be stored locally in a prod environment (this is the case where the returned URL is `"your local fides_uploads folder"` instead of a URL), just included this for extra clarity.